src/forms/inputcounter/InputCounter.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+import * as React from "react";
+import * as utils from "../../utils";
+import {Icon} from "../../icon";
+import {Input} from "../input/Input";
+
+type InputCounterProps = {
+    className?: string;
+    maxValue?: number;
+    minValue?: number;
+    defaultValue?: number;
+    name?: string;
+    errMsg?: string;
+    width?: string;
```

Comment:
instate of only width it is better if we give control of whole style by using 'style' prop.

Reply:
As I need to allow only width to change, I have not give style as prop.